### DISCUSSION TITLE
Phase 7 - The Plan

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#554] [Announcements - Phases]
Time to start formalizing a list of what we want to be in phase 7:

- ~~publisher medium and feeds #579 (enhancement)~~
- authorization #534
- ~~keysend address #576 (enhancement)~~
- ~~chat #502~~
- tags #590 
- ~~simplified license list (enhancement) #603~~
- ~~course medium #577~~ 
- improve the description of fees and splits #596 (clarification)
- ~~title attribute on remoteItem (enhancement) #629~~
- ~~add item as parent for contentLink tag #614~~

**Comment by ryan-lp** [comment]
Transcripts.

* #519 
* #452 
* #484 (although may be superseded by the first issue)
* #458 (mainly the labelling of the generator, disagree with the optout)

A "proper" solution to dynamic ad insertion that does not break timestamp-based search engines (and other apps) would also be very nice. But realistically this is probably not something we would be able to reach agreement on in Phase 7. I have written a proposal here: #556

**Comment by samsethi** [comment]
Hi 

In Phase 7 I would like to see the following please.  

<Podcast:VerifyOwnership> 
<Podcast:Activity> previously Events
<Podcast:Chat>
<Podcast:Country> - to counter balance <Podcast:Location> Podfans uses 'Country' to find podcasts from a particular country. e.g Home Grown Hits.  It allows users/fans to change the country tag and see podcasts from a different country/language

PodFans Search Filters showing Country. 
<img width="1394" alt="Podfans___Search_for_Podcasts" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/b87ade9f-a8e6-4e3c-82ce-fe460feab196">

PodFans Admin view enabling Podcasters to select their country. 
<img width="1124" alt="Podfans___Dashboard" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/e9e1b48a-6689-4fe8-b4d9-4aa71e235855">

**Reply by Dwev** [reply]
I support the `country` and `chat` tags.

**Reply by Dwev** [reply]
Actually, I changed my mind on country, based on the suggestion of @theDanielJLewis to add an attribute to the location tag like "origin=true" or similar. I think this addresses Sam's use case, has more functionality, and keeps the namespace simpler.  Discussion here:

https://github.com/Podcastindex-org/podcast-namespace/discussions/616#discussioncomment-8684699

**Reply by picard102** [reply]
Agreed, making the location tag more flexible is the path forward here.

**Reply by samsethi** [reply]
Sadly I disagree but that's ok. Country level is a simple marker.  If location was used then maybe Marlow, UK covering town/city and country would work.  As I have said before I think the original location was badly named. James intended it to be the location of the podcast subject. i.e Talking about the Vatican in Rome when recording in London. I think the location tag as is could be renamed to subject and fit the original criteria.  Then the location tag could be adapted to include town/city and country with a geo location being a mandatory not optional field.

**Reply by jamescridland** [reply]
I would like to explore, again, the `podcast:location` tag to make it bulletproof and clear, and capable of covering _both_ use-cases of "editorial subject" and "recording location". Given the low amount of take-up, I think it's easy to fix this in a way that works for both.

Particularly, I *really* believe that we shouldn't be just using lat/lon or, worse, a random piece of text in here. In order for this to work for podcast clients, we should change the code to **mandatorily** be adding some form of ID about the place discussed.

R3727240 is the OpenStreetMap code for Marlow, Buckinghamshire. https://www.openstreetmap.org/relation/3727240 
Q508998 is the Wikidata ID for Marlow, Buckinghamshire. https://www.wikidata.org/wiki/Q508998

Both communicate that this is a town in England with a defined boundary; OSM has a link to the Council website. Wikidata has a nice photograph, says it's in a county called Buckinghamshire, that it's twinned with Budapest, links to people born in Marlow and deaths, and links to all kinds of places across the web to help. We learn that in Farsi, it's spelt 	مارلو، باکینگهام‌شر and in Russian, it's Марлоу. Both link to each other.

51.566667, -0.766667 is a lat/lon that *could* be used for Marlow, or *could* be used for Lock Island, which is where it actually points, which is an island in Marlow in the middle of the River Thames. But it tells us nothing other than a lat/lon point. It's useless when it comes to giving more advanced queries like "give me podcasts about all towns in Buckinghamshire". Or, indeed, "give me all podcasts about islands". We don't know what the podcast is about, or where it's from, if we only give a lat/lon. It's a bad thing.

Instead of hurtling into another tag with a hand-written random piece of text, could we please spend the time to get this properly and correctly specified, within the right tag, so that we can move forward? I am absolutely fine with ripping up `podcast:location` and starting again: but we should be able to fix this in a good way which is flexible for the future, rather than a database-filling exercise.

**Reply by picard102** [reply]
Agreed, some sort of standardized way to specify data would be ideal. Not entirely familiar with OSM, but would it also have id codes for more granular locations like Lock Island, or is the highest level city wide?

**Reply by jamescridland** [reply]
@picard102 Thanks!

OSM goes down to individual buildings.

* W32837373 is Lock Island - https://www.openstreetmap.org/way/32837373
* W589058696 is a building on Lock Island in the middle of the lock itself - https://www.openstreetmap.org/way/589058696
* N5620566294 is the bench directly opposite that building on the opposite side of the lock - https://www.openstreetmap.org/node/5620566294 - and we know that it has a backrest.

Wikidata might be easier for developers to work on, though OSM<->Wikidata links do exist as well. Wikidata certainly won't link to the bench.

What I find exciting about OSM is that a) it's entirely open, and user-edited; b) the wealth of data in it could lend itself to some really nice user experiences. As one example: while 51.5710860, -0.7664026 just points to a spot in Marlow, https://www.openstreetmap.org/node/7159380269 tells us its a railway station, and that it's wheelchair accessible. "Please give me a list of podcasts about railway stations that I can get onto in my wheelchair" is perfectly possible with this dataset. (As, with an additional field, "Please give me a list of podcasts recorded in Buckinghamshire").

**Reply by jamescridland** [reply]
> "Please give me a list of podcasts about railway stations that I can get onto in my wheelchair" is perfectly possible with this dataset.

Enclosed is the query that would give "railway stations that I can get onto in my wheelchair" in England, just to make that point nicely. Try doing that with a) raw text, b) lat/lon

<img width="1360" alt="Screenshot 2024-04-10 at 11 47 04 AM" src="https://github.com/Podcastindex-org/podcast-namespace/assets/231941/dcf885e3-478a-43f7-90a0-18964b4a437d">

**Reply by Dwev** [reply]
I love this! The examples
you provide really show the potential for using OSM, as it pertains to getting more data about the location. I really like the idea of using this feature like a guided audio tour, where a supporting player could play chapters based on location, and show a tour route around a town or city. 

One drawback maybe, is that OSM, while open, is not a standard. Lat/Lon is a standardized location value that can be used with any service (no one owns it), but the OSM code can only be used with OSM (although I'm sure you could use an OSM code to lookup a lat/long.) Do you see this as a problem?, @jamescridland ?

**Reply by jamescridland** [reply]
Thank you!

The current podcast:location spec suggests:

```xml
<podcast:location
  geo="geo:-27.86159,153.3169"
  osm="W43678282">
Dreamworld (Queensland)
</podcast:location>
```
...but says that the OSM ID is optional. I'd like to suggest that BOTH the "geo" and a placeID of some sort, are both mandatory. Up to us to agree on the placeID.

Without an OSM id, or a Wikidata ID, or something else, we can't do anything other than plot points on a map. For the downsides of that, see https://maps.fm which is an excellent service, but doesn't know when a podcast is about the Eiffel Tower, or Paris, or all of France. With an OSM ID, it *would*, unambiguously.

OSM is entirely open, has global coverage, and the full database is downloadable. It powers hundreds of mapping products worldwide, including (I'm led to believe) your Amazon delivery driver. I'm not sure of the definition of a "standard"; but OSM is open, and is freely available with a clear attribution licence. Equivalent placeIDs from Google Maps are also available, but you probably don't want to be using a proprietary service like Google Maps.

**Reply by Dwev** [reply]
Yeah, maybe "standard" was the wrong word. I meant that you can plug lat/lon into almost any mapping service and they should all return the same location.  OSM can only be plugged into OSM.  Making both mandatory will also result in additional workload for the podcaster, where they will need to find the lat/lon _and_ look up OSM to get the code (and maybe this isn't a problem.)

Is there a middle-ground where either can be mandatory, but at least one is (preferably OSM)? 

We can always derive a lat/lon from the OSM, so that would avoid some redundancy and extra work for the podcaster. The app dev can use the OSM code to get the lat/lon if that's all they want to show, but they can also pull the other metadata if they choose.

**Reply by jamescridland** [reply]
> Is there a middle-ground where either can be mandatory, but at least one is (preferably OSM)?

Our current middle-ground is that **lat/lon is mandatory, but OSM isn't.** I think that was a mistake for the reasons given above - it encourages podcast hosting companies to just use any old lat/lon provider, and leaves us without any detail on the location. I'd like to suggest that **both lat/lon and (a placeID) is mandatory**.

If we've already bullied a podcast hosting company into getting an OSM ID in the first place, I think it's trivial to get the lat/lon to also populate that field: so a mandatory lat/lon comes free with a mandatory OSM ID. To retain a lat/lon in the tag does make it much easier for a simple podcast player to plot lat/lons on a map, without the overhead of looking up millions of OSM IDs - but by ensuring a placeID is in the tag, that allows us to achieve much, much more at relatively low complexity for those that want to work on OSM integration.

If we rebuild the `podcast:location` tag, I'd much rather make it something like:

```
<podcast:location>
<podcast:place rel="aboutplace" geo="geo:-27.86159,153.3169" osm="W43678282">Dreamworld (Queensland)</podcast:place>
<podcast:place rel="recordedin" geo="geo:51.5710860,-0.7664026" osm="N5620566294">The Podcast Bench, Marlow</podcast:place>
</podcast:location>
```

... which would allow us to retain backwards compatibility should we wish. The use of a new `podcast:place` would allow multiple values: so this show was recorded in two places simultaneously.

```
<podcast:location>
<podcast:place rel="aboutplace" geo="geo:-27.86159,153.3169" osm="W43678282">Dreamworld (Queensland)</podcast:place>
<podcast:place rel="recordedin" geo="geo:51.5710860,-0.7664026" osm="N5620566294">The Podcast Bench, Marlow</podcast:place>
<podcast:place rel="recordedin" geo="geo:34.043056,-118.267222" osm="W428021739">The Crypto Arena, Los Angeles</podcast:place>
</podcast:location>
```

This thread is wholly the wrong place to be doing this work in, of course!

**Reply by Dwev** [reply]
Re-reading the spec for location, lat/lon is not mandatory either.  Both the `geo` tag and the `osm` tag are "recommended".

I'm not sure I see the benefit of a nested new tag, as opposed to multiple location tags with an additional context attribute (`rel='subject'` for the content location, `rel='origin'` or `rel='production'` for recording location or otherwise as described by @theDanielJLewis [here](https://github.com/Podcastindex-org/podcast-namespace/discussions/616#discussioncomment-8684699).)

**Reply by jamescridland** [reply]
Fine by me if we don't need a new nested tag.

I'm suggesting that the `geo` and `osm` tag are mandatory. Otherwise, we're really filling a database with un-normalised rubbish, which really doesn't give us any advantages towards getting the best out of this tag.

I'm also absolutely cool with documenting ways to get a "country" out of a lat/lon pair; which ought to be a simple and easy transformation.

**Reply by Dwev** [reply]
Making them mandatory removes the ability to specify a fictional location like "Hogwarts", as shown in the spec.  I don't know how many people are actually using this ability, but I'm wary about reducing existing functionality.

As you alluded to, the question here is whether we support a new `country` tag as proposed by @samsethi and @benjaminbellamy or whether that functionality can be handled by amending the existing `location` tag as proposed by @theDanielJLewis.

**Reply by jamescridland** [reply]
"mandatory in the case of a real location"

I'd really like to avoid cruft. A country tag offers *so* little functionality, and the country is available through this location tag which has so much possibilities.

**Reply by Dwev** [reply]
Github discussions has a poll function.  Maybe we can set a guideline for adoption where we can poll for decisions like this, with a minimum number of votes to pass (or have certain "super" voters like @daveajones that carry more weight).  It's difficult at the moment to get a clear picture of who supports a tag and who doesn't, and maybe a poll can be useful for this?

**Comment by theDanielJLewis** [comment]
I'd still like consideration for the proposals for notes and other rich data in chapters (which seem increasingly pointless to be a separate file with everything else we're cramming in the RSS feed). This would make Podcasting 2.0 chapters actually _better_ for the audiences instead of merely a separate file that benefits almost no one differently after publishing. #400

**Comment by ryan-lp** [comment]
I see the Phase 7 list in the top post has grown, although it is not based on any of the suggestions above. Was their inclusion perhaps based on upvotes of the respective proposals? If so, I would like to draw attention to the following which has the next highest votes behind those already included:

* #538

There might be some bias within this developer community toward Bitcoin that is not representative of the larger world, and this may be reflected by suggestions that we can solve the underlying problem behind my proposal by still using Bitcoin under the hood and converting dollars into Bitcoin when topping up a wallet. But that's a completely separate idea and is not what I'm proposing here. The proposal is to simply add plain old PayPal as one of the payment types in addition to Bitcoin. Is there any reason why we can't consider that? There's no mystery behind it, listeners would just use a familiar currency and a familiar payment method that they may have greater trust in, and which they may indeed already be signed up for, lowering their friction to participate in V4V. Having choice is a good thing, and this is a choice that is currently lacking in V4V.

https://github.com/Podcastindex-org/podcast-namespace/discussions/538#discussioncomment-6958280 gives a concrete example of what the XML would look like:

```xml
<podcast:value type="paypal" suggested="0.05USD" minimumPayout="5.00USD">
    <podcast:valueRecipient address="paypal.me/person1" split="50"/>
    <podcast:valueRecipient address="paypal.me/person2" split="50"/>
</podcast:value>
```

No SDK is needed for the app developers to implement this, only logic to keep track of listen time and once it reaches the minimum payout, prompt the user to click the PayPal button which just launches the URL `https://paypal.me/RECIPIENTNAME/{amount}{currency}` constructed from the recipient address.

I respect what everyone is trying to do with crypto but let's just step back a bit and notice that listeners will probably welcome having a familiar option to go along with other payment options.

**Reply by jamescridland** [reply]
Agree.

(I'd separate currency from the amount; so `suggested="0.05" minimumPayout="5.00" currency="USD"` just to make it rather easier for developers; currency is optional and defaults to USD unless set).

**Reply by samsethi** [reply]
V4V doesn't stipulate Bitcoin as the only currency.  The reason for not using FIAT payment gateways like Paypal, Stripe etc has been the prohibitive fees and the minimum transaction values. I agree that using a FIAT currency which is well known reduces the barrier to entry. I have high hopes that Strike will enable this with their wallet that enables fiat payments of lightning.  Fountain have already implemented the Strike Wallet as an option and TrueFans is not too far behind.

**Reply by ryan-lp** [reply]
> V4V doesn't stipulate Bitcoin as the only currency.

But it does still require a change to the spec in order to support payment gateways that have minimum viable transaction values. Since the original design was only tested on crypto, it requires actually testing on other payment gateways to see where the original design was lacking.

> I have high hopes that Strike will enable this with their wallet that enables fiat payments of lightning. Fountain have already implemented the Strike Wallet as an option and TrueFans is not too far behind.

The proposal is to make a change to the spec in order to support PayPal. Is "fiat payments of lightning" a reason not to support PayPal as a gateway option?

**Reply by nathangathright** [reply]
> No SDK is needed for the app developers to implement this, only logic to keep track of listen time and once it reaches the minimum payout, prompt the user to click the PayPal button

If we’re expecting listeners to visit multiple in-app web browser windows sequentially to pay out all the V4V recipients manually, then as a UX designer, I’d be extremely skeptical that listeners will follow through past the 1st or 2nd. In my opinion, that doesn’t go far enough to achieve the goal of lowering the friction to participate in V4V. At that point, this might as well become an enhancement to the `<podcast:funding>` tag to support episode-level splits.

IMO, supporting In-App Purchases is critical to wider adoption beyond the users willing to jump through Bitcoin hoops. If that can be addressed by integrating a PayPal SDK into the apps to pass funds directly from listeners to PayPal, then I’d love to see that. But I don’t expect app developers to be eager to keep adding more and more SDKs to accommodate Cash App, BuyMeACoffee, Ko-fi, Stripe, etc.

**Reply by ryan-lp** [reply]
Granted, such a payment method is not well suited to having 15 splits, but it would at least be well suited to podcasts that have a single split, which is the usage pattern that podcasters using PayPal, BuyMeACoffee, ... already fall into. I could also see it working for 2 splits. But primarily this would add the benefit of streaming value according to listen time which is something that `<podcast:funding>` doesn't do.

As a listener, this is actually also the UX that I would feel more comfortable using, since I at least get a chance to confirm and approve the payment and perhaps see a summary of my listen time before it goes through. I want that.

And as an app developer, it is the UX that I would feel more comfortable building, since I feel it might reduce my liabilities if I had a bug in the payment logic that automatically and continuously sends out lots of payments to lots of splits without user confirmation.

And of course, for the podcaster, the choice really should have been theirs all along.

But until the spec actually provides even the "option" for something like PayPal to coexist, we can't claim that the spec has always been flexible enough to support other payment methods, and that the fact that nobody has done it yet proves that nobody wants it. The way to find out whether anybody wants it is not to see what developers want to push, since a disproportionate number of developers here want Bitcoin. The way to find out what people want is to conduct an unbiased survey among podcasters.

**Reply by nathangathright** [reply]
I took a crack at how I’d implement something like this in StenoFM with the constraint of designing an experience for a single split. Let me know if that’s along the lines you were thinking. 
![image](https://github.com/Podcastindex-org/podcast-namespace/assets/3632071/7c91a9b4-685f-43e8-9f49-989c8c7a5d71)

This could be implemented today with the existing funding tag. Developers can append the amount as URL params to PayPal URLs, but it gracefully falls back to a suggested amount for listeners to type in themselves with any other funding tag destination.

However, I wouldn’t say that fulfills the ethos of V4V without "paying all the creators in the value chain." For that, I think we’d need something like a version of https://conshax.app/support that accepts PayPal and handles the splits behind the scenes without making the listener jump through several hoops.

**Reply by ryan-lp** [reply]
The mockups look good, and I appreciate your giving time to think about payment method choice as an app developer. Although just to clarify what the proposal is:

> Let me know if that’s along the lines you were thinking. This could be implemented today with the existing funding tag.

The proposal was for streaming PayPal payments according to listen time (i.e. the PayPal equivalent of streaming sats). This type of functionality falls under the `podcast:value` tag and it requires a change to the spec in order to support a `minimumPayout` because we want to have infrequent batched payments rather than frequent micropayments.

From a UX perspective, the listener would be listening to a podcast for some number of days or weeks before eventually reaching the `minimumPayout`. Then the listener opens their podcast app one day and they are prompted to send a value payment to the creators with a summary showing what their listen time was over the past amount of time. Listeners give consent to every payment based on visibility of their listen time, but can bump up that minimum payout threshold to send through accumulated payments less frequently.

**Reply by nathangathright** [reply]
Oops, that slipped my mind somewhere along the way. I took another crack at it, and came away with more questions. But first the mockup!

![image](https://github.com/Podcastindex-org/podcast-namespace/assets/3632071/ff70db7c-8550-4956-8fb7-6244d855d3e1)

1. I went with a piggy bank icon to compliment the "accumulating" metaphor. I’d love it if you have any other suggestions.
2. Invoking this modal from the Now Playing screen would show only the Value Recipients from the current episode, but presumably, apps are globally keeping track of "listen time per recipient" across multiple shows. (So, a tiny split in _this_ episode may have accumulated the highest payout from _other shows_ they were a part of.)
3. This mockup assumes the user is setting one global stream rate and one global minimum payout. The user could update those values from this modal and see the totals and percentages update in real-time.
4. Setting a Minimum Payout in every feed feels unnecessary. Apps will make it a user setting anyway, and sensible defaults can be suggested in the community. 
5. If a feed has multiple value blocks, I presume the app will pick one and ignore the others based on their preferences for specific values of the `type` attribute (lightning|paypal|~webmonetization~). Do you still think they should be grouped by currency? If not, should `currency` be set per Recipient instead?
6. If the payments to this value block are all manual, is there any reason to limit the recipients to a single payment platform? Why not mix all the manual options? (If so, presumably, links like https://getalby.com/p/nathang would also be acceptable.)

**Reply by ryan-lp** [reply]
Seeing it all fleshed out, you've raised some very sensible points. Definitely, Minimum Payout should be a general user preference, not a podcast-specific thing.

Yes I do still think it makes sense to group value blocks by currency. In this case, a solution might look as follows, where each recipient may offer multiple links with a URL schema that indicates where parameter substitutions can go (if available):

```xml
<podcast:value type="manual" suggested="0.05" currency="USD">
    <podcast:valueRecipient split="50">
        <podcast:paymentAddress>https://paypal.me/person1/{amount}{currency}</podcast:paymentAddress>
    </podcast:valueRecipient>
    <podcast:valueRecipient split="50">
        <podcast:paymentAddress>https://paypal.me/person2/{amount}{currency}</podcast:paymentAddress>
        <podcast:paymentAddress>...........................................</podcast:paymentAddress>
    </podcast:valueRecipient>
</podcast:value>
```

**Reply by nathangathright** [reply]
Maybe I’m misunderstanding what problem grouping value blocks by currency is trying to solve. If a feed has multiple `type="manual"` value blocks with different currencies, I presume apps will prioritize one of them (ideally in the listener’s currency of choice to minimize conversion fees?) and ignore any recipients in the other(s) to avoid dealing with any discrepancies between splits/recipients in different blocks. 

For example, if person1 and person2 can accept GPB & USD, but person 3 can only accept USD, listeners in the UK will only see the recipients from the `currency="GPB"` block and won’t be given the option to support person3.
```xml
<podcast:value type="manual" suggested="0.05" currency="GBP">
    <podcast:valueRecipient split="50" address="https://foo.co.uk/person1/{amount}{currency}"/>
    <podcast:valueRecipient split="50" address="https://foo.co.uk/person2/{amount}{currency}"/>
</podcast:value>
<podcast:value type="manual" suggested="0.05" currency="USD">
    <podcast:valueRecipient split="40" address="https://bar.com/person1/{amount}{currency}"/>
    <podcast:valueRecipient split="40" address="https://bar.com/person2/{amount}{currency}"/>
    <podcast:valueRecipient split="20" address="https://bar.com/person3/{amount}{currency}"/>
</podcast:value>
```

If we complicate it further with multiple payment addresses per recipient, what’s unacceptable about this approach?
```xml
<podcast:value type="manual" suggested="0.05">
    <podcast:valueRecipient split="40">
        <podcast:paymentAddress currency="GBP" address="https://foo.co.uk/person1/{amount}{currency}"/>
        <podcast:paymentAddress currency="USD" address="https://bar.com/person1/{amount}{currency}"/>
    </podcast:valueRecipient>
    <podcast:valueRecipient split="40">
        <podcast:paymentAddress currency="GBP" address="https://foo.co.uk/person2/{amount}{currency}"/>
        <podcast:paymentAddress currency="USD" address="https://bar.com/person2/{amount}{currency}"/>
    </podcast:valueRecipient>
    <podcast:valueRecipient split="20">
        <podcast:paymentAddress currency="USD" address="https://bar.com/person3/{amount}{currency}"/>
    </podcast:valueRecipient>
</podcast:value>
```

**Reply by ryan-lp** [reply]
Having a common currency is a prerequisite for computing splits, and in practice the feed owner gets to decide which currency, while other value recipients would be invited into the value chain using a currency that the feed owner decided on. The problem that is solved in grouping by currency rather than grouping by payment gateway is that since there are many alternative payment gateway options for the same currency and since the only prerequisite to calculating a split is the common currency, this opens up more flexibility to allow more recipients to participate in a value split even though they may have accounts through different payment gateways.

Regarding your last example, I suppose you are wondering whether you can make this even more flexible by mixing all currencies together into a single block. I suppose you can, although in practice if every value recipient wants to choose a different currency, and you leave it up to the listener to choose the currency that has the lowest conversion fee, then only one lucky recipient gets any value, and perhaps not even the feed owner gets any value. There needs to be some agreement on currency in order for splits to work, and I think in the end it is probably up to the feed owner to make the decision. But technically speaking, there is nothing wrong with mixing multiple currencies into one block - as you've shown, it can be viewed as simply a more concise way of representing exactly the same behaviour as having multiple value blocks, except you don't have to repeat the split ratios again and again, which is a benefit. Maybe, then, you could even mix bitcoin payments into the same structure as well and reuse those same split ratios.

**Reply by samsethi** [reply]
This is awesome.  Now we have both SATs and Fiat payments in TrueFans I need to read this several times to get my head around how this might work.  On all episodes, we show both the SATs value = SATs per minute x length of episode but we also show that converted into the local currency.  The challenge is of course if paid for in Fiat the payment gateway takes 3% but that is not the case with P2P SAT payments however I can see the benefit of offering a FIAT payment option.

**Reply by samsethi** [reply]
@ryan-lp 

> From a UX perspective, the listener would be listening to a podcast for some number of days or weeks before eventually reaching the minimumPayout. Then the listener opens their podcast app one day and they are prompted to send a value payment to the creators with a summary showing what their listen time was over the past amount of time. Listeners give consent to every payment based on visibility of their listen time, but can bump up that minimum payout threshold to send through accumulated payments less frequently.

The only concern as an app developer is what if the listener consumes a vast amount of content and implies they are willing to pay for it but when it comes to actually making the final payment they decide not too or don't have sufficient funds.  Taking payments per minute using SATS enables us to prevent this use case.  Of course the user can set the V4V to zero and not pay at all. I am wondering if we can take a prepayment in advance.  So the user has $10 on account and then when then minimum payment is met we can take the $5 and remind them to top up their account?  In effect we are doing this with SAT's wallets.  Users are storing 10K SATs in their wallet and then choosing to stream/boost using those SATs.  Food for thought

**Reply by Dwev** [reply]
I think fiat works for boosts, but for streaming I think it's better to convert fiat into sats, since it can be such a small amount per minute.  Managing streaming fiat could be tricky.

**Comment by picard102** [comment]
I'd like to see the person tag improved upon. 
https://github.com/Podcastindex-org/podcast-namespace/discussions/498

As well as the social tag on the channel level be implemented.
https://github.com/Podcastindex-org/podcast-namespace/discussions/559

**Reply by picard102** [reply]
Would love to see person tag improved before PSP makes a big push to get wide support for it.

**Comment by nathangathright** [comment]
Some proposals for phase 7 consideration:
* [`<podcast:follow>`](https://github.com/Podcastindex-org/podcast-namespace/discussions/609)
* [`<podcast:contentLink>` Enhancement](https://github.com/Podcastindex-org/podcast-namespace/discussions/614)

**Reply by nathangathright** [reply]
[Buzzsprout](https://github.com/Podcastindex-org/podcast-namespace/discussions/589#discussioncomment-8329874), [RSS.com](https://github.com/Podcastindex-org/podcast-namespace/discussions/609#discussioncomment-9086113), [Podhome](https://github.com/Podcastindex-org/podcast-namespace/discussions/609#discussioncomment-9087344), [TrueFans](https://github.com/Podcastindex-org/podcast-namespace/discussions/609#discussioncomment-9087043), and [Episodes.fm](https://nathangathright.com/introducing-episodesfm/#verifying-additional-apps) have expressed interest in `<podcast:follow>`. Is there any chance it’ll make it into Phase 7?

**Reply by Dwev** [reply]
I'm happy to execute it for Castopod as well, if @yassinedoghri is cool with it...

**Comment by samsethi** [comment]
[podcast:host](https://github.com/Podcastindex-org/podcast-namespace/discussions/625) is a no brainer and easy to implement by hosts with no user input.  I could also get it pushed through the PSP 1.1 tags which is the next batch of tags to universally support.

**Reply by Dwev** [reply]
I agree that the intent of this tag is super easy to implement.

Perhaps I'm being too pedantic, but my only caveat is that maybe `platform` or even `podcast:generator` is a better name, as `host` could mean the media host, web host, or even the person hosting the podcast!

Platform/generator can cover self-hosted podcast RSS generator apps, as well as all the hosted apps.

e.g. I use AWS to host my website, and Cloudflare as a CDN for my media, but I'm using a self-hosted version of Castopod.
The web **host** is **AWS**. The URL my **media** is served from resolves to **Cloudflare**. The **platform/generator** is **Castopod**.

For analytics, all this can be interesting, although my understanding is that we mostly care about the platform/generator.

**Comment by samsethi** [comment]
Hi @daveajones 

In an attempt to help you before Phase 7 closes.  I have tried to collate all conversations about new/proposed tags with subsequent links into this one post. 

1. **[Publisher Feeds](https://github.com/Podcastindex-org/podcast-namespace/discussions/579)** look good to go.  Podnews Daily is already supporting them.  So are RSSBlue, Wavlake and TrueFans.  I think Fountain are nearly there too. 

We have also created separate pages for artists and audiobooks.  So if we detect medium=music we create an artist ([publisher](https://truefans.fm/artists)) page as opposed to a podcast ([publisher](https://truefans.fm/publishers)) page. 

2. **Keysend Address** it would be great if we can make this part of P7. 

<img width="1355" alt="TrueFans___Dashboard" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/1064dd60-fcdc-4126-842f-46512fccc81a">

3. **[Verify Tag](https://github.com/Podcastindex-org/podcast-namespace/discussions/534)** I think is dead.  I think all hosts I have spoken to prefer to use TXT to do the verification and most hosts have now added this tag. 

4. **[Chat](https://github.com/Podcastindex-org/podcast-namespace/discussions/502)** 
I think this Chat tag will be soon superceded.  Adding an external place for conversation feels like the wrong direction for apps.  We support this for Podcasting 2.0 but see no other call for this external chat link, TrueFans will be launching co-listen shortly to replace external Chat links inside the app. 

<img width="1357" alt="TrueFans___Dashboard" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/653c2f65-c449-4556-8fc7-837c0d76f64c">

5. **[Category Tags](https://github.com/Podcastindex-org/podcast-namespace/discussions/590)**

I am not sure why we need this tag? We already have categories and keywords.  Categories are defined taxonomies.  We have recently added a new Music taxonomy from Wavlake and TrueFans show this when we detect meduim=music.  We can collectively develop a new taxonomy for other categories like audiobooks, films, courses etc. 

Keywords are podcast creator ad-hoc lists.  And if that doesn't work we have the free-form description field for show notes.  

<img width="892" alt="TrueFans___Dashboard" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/ce3fb41c-a586-48f5-8149-ebba5c7c722f">

6. **Course medium**
Podhome and Truefans already support this extension to the medium tag.  Can we also make audiobooks, [radio](https://github.com/Podcastindex-org/podcast-namespace/discussions/567), video and films part of this phase. The tag is simple to implement but apps then need to design how they present the different types of medium.  e.g change episodes to lessons if the medium=courses. 

<img width="1152" alt="Fullscreen_07_05_2024__11_32" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/9ffa4b89-c65a-4a56-9fd5-19bb455e3640">

7. [Podcast License](https://github.com/Podcastindex-org/podcast-namespace/discussions/603) 
This tag was originally confusing but in TrueFans we have just used the 6 Creative Commons licenses and added the new Podcast License to cover music. 
<img width="886" alt="TrueFans___Dashboard" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/c97f30d8-ee5b-4fab-975f-ddcba9fe7c9e">


Other tags for consideration based on GitHub conversations.  

8. Improve the description of **[fees](https://github.com/Podcastindex-org/podcast-namespace/discussions/456)** and **splits** 
9. [Title attribute](https://github.com/Podcastindex-org/podcast-namespace/discussions/629) on remoteItem
10. [Podcast: follow](https://github.com/Podcastindex-org/podcast-namespace/discussions/609) 
11. [Podcast: contentLink](https://github.com/Podcastindex-org/podcast-namespace/discussions/614) (enhancement)
12. [Podcast: person](https://github.com/Podcastindex-org/podcast-namespace/discussions/498) (enhancement) i.e multiple roles.  In TrueFans we have already enabled this.  So a person can be a singer and songwriter or a person can be host and producer. It is also essential we can extend the person taxonomy as it is heavily focused on the podcast roles and not music or other roles.  We have extended the Person Taxomony in TrueFans to include songwriter, singer, AI, DJ (for musicasts), VJ (ValueJocks or VideoJocks) etc. 
13. Podcast: location (enhancement) - @jamescridland has suggested that GEO or OSM location corodinates is not optional but mandatory.  This was in response to the [podcast:country](https://github.com/Podcastindex-org/podcast-namespace/discussions/616) tag proposal. 
14. [Social Interact Tag](https://github.com/Podcastindex-org/podcast-namespace/discussions/634) (enhancement) added to Channel
15. [Podcast Pinned](https://github.com/Podcastindex-org/podcast-namespace/discussions/570) seems to fall between the cracks 
TrueFans enabled this at the same time we enabled Podrolls.  In our admin dashboard we enabled podcast creators to have their trailer episode as the default pinned episode or to choose another episode to feature.  Adding this to the RSS feed and host to build a similar UI would be simple.  

![Cursor_and_TrueFans___Shows___Podnews_Daily](https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/eacc712a-2020-43fc-983f-125a7420f18c)

I think this is everything @daveajones  I can see on GitHub for Phase 7. Hope this helps. 

**Phase 8 might include** 

1. Podcast:activity - this is a JSON file link to a users activitystream much like chapters. Then users can choose to publish their activity via ActivityPub to client as well as the socialInteract Tag. This is the big tag for the next phase.

2. [Single v Album ](https://github.com/Podcastindex-org/podcast-namespace/discussions/578)
3. [Auto Downloads](https://github.com/Podcastindex-org/podcast-namespace/discussions/626)  
4. [Podcast: Public v Private ](https://github.com/Podcastindex-org/podcast-namespace/discussions/627)
5. [Podcast: Host v Generator](https://github.com/Podcastindex-org/podcast-namespace/discussions/625)
6. [Podcast: valuePercentage](https://github.com/Podcastindex-org/podcast-namespace/discussions/606)

I hope I captured everything but if anyone can spot other things to add for Dave's consideration please list them below.
